src/extension/host-binary/cmd/main.go

```diff
@@ -0,0 +1,84 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"github.com/docker/labs-ai-tools-for-devs/pkg/client"
+	secretsapi "github.com/docker/labs-ai-tools-for-devs/pkg/generated/go/client/secrets"
+	"github.com/docker/labs-ai-tools-for-devs/pkg/paths"
+	"os"
+	"os/signal"
+	"syscall"
+
+	"github.com/spf13/cobra"
+)
+
+func main() {
+	ctx, closeFunc := newSigContext()
+	defer closeFunc()
+	paths.Init(paths.OnHost)
+	cmd := AddSecret(ctx)
+	if err := cmd.Execute(); err != nil {
+		fmt.Println(err)
+		os.Exit(1)
+	}
+}
+
+func newSigContext() (context.Context, func()) {
+	ctx, cancel := context.WithCancel(context.Background())
+	s := make(chan os.Signal, 1)
+	signal.Notify(s, syscall.SIGTERM, syscall.SIGINT)
+	go func() {
+		<-s
+		cancel()
+	}()
+	return ctx, cancel
+}
+
+func newApiClient() (client.ApiClient, error) {
+	p, err := paths.GetSecretsApiSocketPath()
+	if err != nil {
+		return nil, err
+	}
+	return client.NewApiClient(p), nil
+}
+
+type addOptions struct {
+	Name  string
+	Value string
+}
+
+func AddSecret(ctx context.Context) *cobra.Command {
```

Comment:
This should be good enough for the start. Just run `host-binary add --name <secret-name> --value <secret-value>`. (you can adjust to your needs later if you want)